### DISCUSSION TITLE
500 : Internal Server Error

### DISCUSSION THREAD

**Discussion by xavierhill203** [cooperrc/computational-mechanics#79] [Q&A]
I am trying to log on to the JupyterHub but it states this error. Does anyone know how to fix this error or get around it?

**Comment by cooperrc** [comment]
hmm, I'm getting that one too for some reason [8am Tue 9/06/22]. I'll take a look.

**Reply by cooperrc** [reply]
**Update 20220906 915am**: 

There was a storage issue on the server that wouldn't let us use any more disk space. I fixed the problem for now, and IT is working on preventing it from happening again.
